docs/configuration.rst

```diff
@@ -253,6 +253,14 @@ The ``TZ`` sets the timezone Mailu will use. The timezone naming convention usua
 
 .. _`TZ database name`: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 
+The ``PROXY_PROTOCOL`` (default: unset) allows the the front container to receive TCP and HTTP connections with
+the PROXY protocol. It can be set to:
+* ``http`` to accept the ``PROXY`` protocol on nginx's HTTP proxy ports
+* ``mail`` to accept the ``PROXY`` protocol on nginx's mail proxy ports
+* ``all`` to accept the ``PROXY`` protocol on all nginx's HTTP and mail proxy ports
+
+This requires to have a valid ``REAL_IP_FROM``.
```

Comment:
We may want to rephrase this and put a big fat warning: setting a broad REAL_IP_FROM can turn you into an open-relay.

We may also want to clarify what the PROXY Protocol is; maybe link https://www.haproxy.org/download/2.6/doc/proxy-protocol.txt ?

Reply:
[Added](https://github.com/Mailu/Mailu/pull/2717/files#diff-68a31565c7f34fbe6ac80bbc0b4287ee3711fa615ce5b973c1f83f285d0d28faR256-R263) the link to the PROXY protocol main document on Github:
> .. `PROXY protocol`: https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt

And [I phrased this](https://github.com/Mailu/Mailu/pull/2717/files#diff-68a31565c7f34fbe6ac80bbc0b4287ee3711fa615ce5b973c1f83f285d0d28faR265-R268) a bit dfferently; 
> This requires to have a valid ``REAL_IP_FROM`` (default: unset). Setting ``PROXY_PROTOCOL`` without setting
``REAL_IP_FROM`` *will not work*. Make sure to set a ``REAL_IP_FROM`` only pointing to IP addresses or networks
that you trust; accepting the ``PROXY`` protocol from untrusted sources is a serious security vulnerability,
allowing a potential attacker to spoof his source address.